### ISSUE TITLE
AI crew not using scooters.

### ISSUE THREAD

**Issue by Cyborgcr** [Regalis11/Barotrauma#1537]
I don't know if it's a bug or not, but the AI crew members when have scooters equipped don't use them. I first found out about this when I was ordered my AI crew member in singleplayer to follow me to ruins. I prepared everything and equipped scooters for both, but the AI didn't use them. Here's a photo from video I did since Github doesn't accept MP4 files.
![Barotrauma 08-Jun-19 11_54_09_Moment(2)](https://user-images.githubusercontent.com/46865336/59144960-e4c2d280-89e5-11e9-9a97-b74e17541e5f.jpg)

**Comment by OverwatchVoice** [comment]
Yeah, they don't know how to use them. Being in the ruins tends to really confuse them, too.

**Comment by itchyOwl** [comment]
Not yet implemented. Thanks for reminding!
